e2e/test/scenarios/admin-2/people.cy.spec.js

```diff
@@ -472,9 +472,8 @@ describe("scenarios > admin > people", () => {
         assertTableRowsCount(PAGE_SIZE);
         cy.findByLabelText("Previous page").should("be.disabled");
 
-        // cy.findByLabelText("Next page").click();
-        cy.findByTestId("next-page-btn").click();
-        waitForUserRequests();
+        cy.findByLabelText("Next page").click();
+        cy.wait("@users");
```

Comment:
RTKQuery refactor fixed an issue where groups were getting reloaded unnecessarily as the user navigated through pagination.